app/flows/check_travel_during_coronavirus_flow/outcomes/_travel_result_layout.erb

```diff
@@ -7,21 +7,23 @@
 
 <div class="govuk-grid-row travel-abroad__section">
   <div class="govuk-grid-column-one-third travel-abroad__reasons">
-    <p class="govuk-body govuk-body-s">Because you said</p>
-    <ul class="govuk-list govuk-list--bullet govuk-body-s">
-      <% data[:reasons].each do |reason| %>
-        <li><%= reason %></li>
-      <% end %>
-      <% if calculator.travelling_with_young_people? %>
-        <li>You are travelling with someone aged 17 and under</li>
-      <% end %>
-      <% if calculator.travelling_with_children.include?("zero_to_four") %>
-        <li>You are travelling with someone aged 4 and under</li>
-      <% end %>
-      <% if calculator.travelling_with_children.include?("five_to_seventeen") %>
-        <li>You are travelling with someone aged 5 to 17</li>
-      <% end %>
-    </ul>
+    <% unless calculator.only_travelling_to_ireland? %>
+      <p class="govuk-body govuk-body-s">Because you said:</p>
+      <ul class="govuk-list govuk-list--bullet govuk-body-s">
+        <% data[:reasons].each do |reason| %>
+          <li><%= reason %></li>
+        <% end %>
+        <% if calculator.travelling_with_young_people? %>
+          <li>You are travelling with someone aged 17 and under</li>
+        <% end %>
+        <% if calculator.travelling_with_children.include?("zero_to_four") %>
+          <li>You are travelling with someone aged 4 and under</li>
+        <% end %>
+        <% if calculator.travelling_with_children.include?("five_to_seventeen") %>
+          <li>You are travelling with someone aged 5 to 17</li>
+        <% end %>
+      </ul>
+    <% end %>
```

Comment:
This "because you said" logic has gotten quite unwieldy, and wrapping the existing `if` statements in and `unless` only compounds that.

Perhaps instead, "Because you said" could be built in the calculator?  A new methods that uses the existing methods, i.e. `travelling_with_young_people?` to return a list of content instead of populating `data[:reasons]`.
Another benefit to this is that the first line of the method could return an empty array if the user is only travelling to Ireland. i.e.:

`return if travelling_to_ireland? && single_journey?` 

This is a bit more work but would make the views easier to manage.

Reply:
this was a bigger bit of work. Done now although tests are failing so I'll fix whatever that is before a rereview